tests/test_helpers.py

```diff
@@ -40,3 +46,122 @@ def generate_random_bboxes(
     areas = (x_max - x_min) * (y_max - y_min)
     bboxes = np.column_stack((x_min, y_min, x_max, y_max))
     return bboxes[areas > 0]
+
+
+@contextlib.contextmanager
+def generate_random_image_folder(width: int = 480, height: int = 360, number_of_images: int = 10) -> Iterator[str]:
```

Comment:
Why not making these functions (`generate_*`) as pytest fixtures?

Reply:
This part is copied from OTX 1.X which is also copied from Geti side. Do you think it's better to change it as fixture?

Reply:
Sure, it's common practice. In addition, you can control the fixture's lifecycle. For example, in this case, you can generate a random image folder once in a session and can reuse it for many test functions. https://stackoverflow.com/a/45833359

Reply:
ok I'll change it.

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3212/commits/e7325279302ee9b77272373c33be053a29148855